File: Android/app/src/main/java/com/etiennebaudoux/clipboardzanager/componentmodel/exceptions/NotNullOrWhiteSpaceRequiredException.java
package com.etiennebaudoux.clipboardzanager.componentmodel.exceptions;

public class NotNullOrWhiteSpaceRequiredException extends RuntimeException {
    public NotNullOrWhiteSpaceRequiredException(String parameterName) {
        super("The value must not be null or white space : " + parameterName);
    }
}

File: ClipboardZanager/Sources/ClipboardZanager.Shared/Exceptions/NotNullOrWhiteSpaceRequiredException.cs
using System;

namespace ClipboardZanager.Shared.Exceptions
{
    public sealed class NotNullOrWhiteSpaceRequiredException : ArgumentException
    {
        public NotNullOrWhiteSpaceRequiredException(string parameterName)
            : base(parameterName, "The value must not be null or white space")
        {
        }
    }
}
